app/models/task_definition.rb

```diff
@@ -373,16 +391,16 @@ def to_csv_row
     TaskDefinition.csv_columns
```

Comment:
Need to export stream as well... and import too.

Plus unit tests to this effect.